topics/epigenetics/docker/Dockerfile

```diff
@@ -18,4 +19,7 @@ ADD topics/epigenetics/tutorials/ /tutorials/
 # install everything for tutorials
 ADD bin/docker-install-tutorials.sh /setup-tutorials.sh
 ADD bin/mergeyaml.py /mergeyaml.py
+ADD bin/datalibrarydownload.sh /datalibrarydownload.sh
 RUN /setup-tutorials.sh
+
+ENTRYPOINT ["/datalibrarydownload.sh"]
```

Comment:
That is a little bit nitpicking, but could you call this script `data_libarary_download.sh`. Its a little bit more readbale.